lib/mux/video/live_streams.ex

```diff
@@ -139,4 +139,55 @@ defmodule Mux.Video.LiveStreams do
   def delete_playback_id(client, live_stream_id, playback_id) do
     Base.delete(client, "#{@path}/#{live_stream_id}/playback-ids/#{playback_id}")
   end
+
+  @doc """
+  Create a live stream simulcast target
+
+  Returns a tuple such as `{:ok, simulcast_target, %Tesla.Env{}}`
+
+  ## Examples
+
+      iex> client = Mux.Base.new("my_token_id", "my_token_secret")
+      iex> {:ok, simulcast_target, _env} = Mux.Video.LiveStreams.create_simulcast_target(client, "aA02skpHXoLrbQm49qIzAG6RtewFOcDEY", %{url: "rtmp://live.example.com/app", stream_key: "abcdefgh"})
+      iex> simulcast_target
+      #{inspect(Fixtures.simulcast_target())}
+
+  """
```

Comment:
Should we provide the doc link in here?

Reply:
good idea, I went through and added the endpoint-specific link for each method